pages/common/pamvalidate.md

```diff
@@ -0,0 +1,9 @@
+# pamvalidate
+
+> Validate Netpbm files.
+> See also: `pamfile`, `pamfix`.
+> More information: <https://netpbm.sourceforge.net/doc/pamvalidate.html>.
+
+- Copy a Netpbm file from `stdin` to `stdout` if and only if it valid; fail otherwise:
+
+`{{command}} | pamvalidate > {{path/to/output.pam}}`
```

Comment:
```suggestion
`{{command}} | pamvalidate > {{path/to/output.pam|pbm|pgm|ppm}}`
```
Maybe it supports PBM, PGM and PPM images too? From the documentation:
> pamvalidate copies a Netpbm image from Standard Input to Standard Output [...]
> [...] To prevent that, pass your PPM input through pamvalidate on its way to pnmtopng. Then, pamvalidate will fail, and consequently the pipeline will fail [...]